mantidproject/mantid
# Main cleanup of Muon v 6.3 release notes

**Description of work.**
The main refactoring the the Muon release notes ahead of release of 6.3. Includes:-

- Removing unused headings and developer warning text
- Reordering of notes where appropriate
- Rewording notes to improve consistency
- Adding links
- Spelling & Grammar check
- Adding an image for new error shading feature

Please note that the Model Fitting notes at the end are currently invisible to users. Apart from moving some notes into this section and rewording all to Model Fitting for consistency no other editing has taken place. This is because we are awaiting approval for release of this feature. These notes will either be revamped or moved to v.6.4 release notes before release.

**To test:**

-     remove build/docs/doctrees
-     build the documents again and check there are no warnings
-     Check that the links work
-     Check the information is still correct
-     Are the images visible and clearly illustrate the new feature/improvement?
-     Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
